src/DIRAC/Core/Utilities/TimeUtilities.py

```diff
@@ -116,13 +127,13 @@ def toString(myDate=None):
       [hour]:[min]:[sec]:[microsec]
       where min, sec, microsec are always positive integers and hour carries the sign.
     """
-    if isinstance(myDate, datetime.datetime):
+    if type(myDate) is datetime.datetime:
```

Comment:
why not `isinstance`?

Reply:
`
In [9]: dt = datetime.datetime.utcnow()

In [10]: isinstance(dt, datetime.date)
Out[10]: True

In [11]: isinstance(dt, datetime.datetime)
Out[11]: True
`

Reply:
I think this can be:

```
-if type(myDate) is datetime.datetime:
-    return str(myDate)
-elif type(myDate) is datetime.date:
+if isinstance(myDate, datetime.date):
```